CMakeLists.txt

```diff
@@ -34,4 +34,8 @@ add_subdirectory(src)
 if (BUILD_TEST_SUITE)
      enable_testing()
      add_subdirectory(tests)
+     endif()
+
+if (CLIENT_COMPILATION_MODE)
+     add_compile_definitions(CLIENT_COMPILATION_MODE)
```

Comment:
> For future use

Use `target_compile_definition()`  in order to set `CLIENT_MODE` only for the client target

Reply:
@aurelenc

Reply:
why not add_compile_definitions only on the client's cmakelists ? @Cavonstavant

Reply:
demo will probably be deleted once the game is successfully networked

Reply:
@aurelenc Yes ofc. We can use target_compile_definitions() to add a def. to specific target